code/game/machinery/fax_machine.dm

```diff
@@ -164,73 +175,59 @@ var/list/alldepartments = list()
 
 	switch(action)
 		if("send")
-			if(tofax)
-				if(target_department == DEPARTMENT_HC)
-					highcom_fax(src, tofax.info, tofax.name, usr)
-					fax_cooldown = 600
-
-				else if(target_department == DEPARTMENT_PROVOST)
-					provost_fax(src, tofax.info, tofax.name, usr)
-					fax_cooldown = 600
-
-				else if(target_department == DEPARTMENT_CMB)
-					cmb_fax(src, tofax.info, tofax.name, usr)
-					fax_cooldown = 600
-
-				else if(target_department == DEPARTMENT_WY)
-					company_fax(src, tofax.info, tofax.name, usr)
-					fax_cooldown = 600
-
-				else if(target_department == DEPARTMENT_PRESS)
-					press_fax(src, tofax.info, tofax.name, usr)
-					fax_cooldown = 600
-
-				else
-					general_fax(src, tofax.info, tofax.name, usr)
-					fax_cooldown = 600
-
-				COOLDOWN_START(src, send_cooldown, fax_cooldown)
-				SendFax(tofax.info, tofax.name, usr, target_department, network, src)
-				to_chat(usr, "Message transmitted successfully.")
-				. = TRUE
+			if(!original_fax)
+				to_chat(ui.user, SPAN_NOTICE("No paper loaded"))
+				return
+
+			copy_fax_paper()
```

Comment:
We probably only need to copy if the copy isn't set. Just be sure to clear the copy when original_fax is removed.